Dima_Serikov/02.Git.Local/02.GIT.Local.md

```diff
@@ -0,0 +1,114 @@
+  993  composer require laminas/laminas-mvc-i18n
```

Comment:
please use code block to display history of commands https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#quoting-code

Reply:
Done